components/causal_ts/src/tso.rs

```diff
@@ -400,9 +377,9 @@ impl<C: PdClient + 'static> BatchTsoProvider<C> {
             .map_err(|_| box_err!("renew response channel is dropped"))
             .and_then(|r| r.map_err(|err| Error::BatchRenew(err)));
 
-        let label = if res.is_ok() { "ok" } else { "err" };
-        TS_PROVIDER_TSO_BATCH_RENEW_DURATION
-            .with_label_values(&[label, reason])
+        TS_PROVIDER_TSO_BATCH_RENEW_DURATION_STATIC
```

Comment:
Do we really need to distinguish the types for the duration?

Reply:
The purpose is as following:

1. To improve the caching algorithm and reduce the chances of TSO "used up", as "used up" will prolong duration of `get_ts()`. The metric can tell us how often "used-up" happens.
2. To judge that how seriously "flush" affects performance of `get_ts()`. It will also help to evaluate the effect that we add `after_ts` to `get_ts()`.